tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_create_new_alarm.py

```diff
@@ -75,17 +74,17 @@ def test_clock_create_new_alarm(self):
         edit_alarm.tap_done()
         self.clock.dismiss_banner()
 
-        self.assertNotEqual(self.clock.alarms[0].time, alarm_time)
+        self.assertNotEqual(alarm_view.alarm_items[0].time, alarm_time)
 
         # turn off the alarm
-        self.clock.alarms[0].tap_checkbox()
-        self.clock.alarms[0].wait_for_checkbox_to_change_state(False)
-        self.assertFalse(self.clock.alarms[0].is_alarm_active, 'user should be able to turn on the alarm.')
+        alarm_view.alarm_items[0].tap_checkbox()
+        alarm_view.alarm_items[0].wait_for_checkbox_to_change_state(False)
+        self.assertFalse(alarm_view.alarm_items[0].is_alarm_active, 'user should be able to turn on the alarm.')
 
         # turn on the alarm
-        self.clock.alarms[0].tap_checkbox()
-        self.clock.dismiss_banner()
-        self.assertTrue(self.clock.alarms[0].is_alarm_active, 'user should be able to turn off the alarm.')
+        alarm_view.alarm_items[0].tap_checkbox()
```

Comment:
Shouldn't there be a alarm_view.alarm_items[0].wait_for_checkbox_to_change_state(True) line here?